src/js/components/Box/README.md

```diff
@@ -751,6 +751,12 @@ string
     xlarge
     string
 }
+{
+  corners: [[object Object]
+[object Object]
+[object Object]
+[object Object]]
+}
```

Comment:
This snapshot doesn't look right and may imply that `corners` value is expecting an array of objects.

Reply:
I'm open to adding a test for that. I'll put one together when I next get a chance.

> This snapshot doesn't look right and may imply that corners value is expecting an array of objects.

I'm not super experienced with these snapshots. I'll see if I can figure out why it is showing an array. Who should I reach out to if I have issues figuring this out?

Reply:
Once you write the tests, you can run `yarn test-update` and it will create the snapshots for you. Push the changed files to this branch and you should be good. CONTRIBUTING.md file should have more details.

Reply:
Thx

Reply:
Hi! Have you addressed this issue so we can continue with the review?

Reply:
It turns out trying to provide both corner and corners throws an error anyway because prop-types doesn't include a definition for it. The functionality still works if a user does it, but tests will not pass. Since there really isn't a good reason for a user to do this I'm not sure it should be officially supported in the prop-types definition of round and we probably don't want a test for it. Is there a case where a user providing both corners and corner wouldn't be a mistake?

Reply:
@ShimiSun  how would you like to move forward?